src/rating/rating.ts

```diff
@@ -5,6 +5,7 @@ import {Component, ChangeDetectionStrategy, Input, Output, EventEmitter, OnInit}
  */
 @Component({
   selector: 'ngb-rating',
+  exportAs: 'ngbRating',
```

Comment:
I exported because the rating hover state is different from the model rate value, and thus needs exposure from the component to display it, unless I'm missing something critical about the outputs - can output values be accessed via $event?